pkg/fn/eval.go

```diff
@@ -17,13 +17,19 @@ package fn
 import (
 	"context"
 
+	"github.com/GoogleContainerTools/kpt/internal/types"
+	fnresult "github.com/GoogleContainerTools/kpt/pkg/api/fnresult/v1"
 	v1 "github.com/GoogleContainerTools/kpt/pkg/api/kptfile/v1"
 	"sigs.k8s.io/kustomize/kyaml/filesys"
+	"sigs.k8s.io/kustomize/kyaml/kio"
 )
 
 type EvalOptions struct {
+	// FnResultList stores the result of the function evaluation
+	FnResultList fnresult.ResultList
```

Comment:
this would need to be pointer (or opts below need to be pointer)

Reply:
updated.